tests/python_on_whales/components/test_compose.py

```diff
@@ -563,7 +651,9 @@ def test_config_complexe_compose():
     docker = DockerClient(compose_files=[compose_file], compose_compatibility=True)
     config = docker.compose.config()
 
-    assert config.services["my_service"].build.context == Path("my_service_build")
+    assert config.services["my_service"].build.context == Path(
+        PROJECT_ROOT / "tests/python_on_whales/components/my_service_build"
+    )

```

Comment:
```suggestion
    assert config.services["my_service"].build.context == PROJECT_ROOT / "tests/python_on_whales/components/my_service_build"
```
It's already a Path, there is no need to convert it to a path again.

Reply:
Whoops 😅  
Fixed. I left a multi-line declaration as I find it more readable.

Thanks a lot for your feedback!